packages/apps/api/src/app.ts

```diff
@@ -1,17 +1,94 @@
-import express, {Application, Router} from 'express';
+import express, {Application, Request, Response} from 'express';
 import bodyParser from 'body-parser';
 
 const app: Application = express();
 
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.urlencoded({extended: true}));
 
-app.post('/forgot-password', function() {
+app.post('/auth', function (req: Request, res: Response) {
+  console.log('Processing [POST /auth]...');
+  res.status(200).json();
+});
+
+app.get('/users', function (req: Request, res: Response) {
+  console.log('Processing [GET /users]...');
+  res.status(200).json([user_one, user_two])
+});
+
+app.post('/users', function (req: Request, res: Response) {
+  console.log('Processing [POST /users]...');
+  res.status(201).json(user_one);
+});
+
+app.get('/users/:user_id', function (req: Request, res: Response) {
+  console.log('Processing [GET /users/:user_id]...');
+  res.status(200).json(user_one);
+});
+
+app.put('/users/:user_id', function (req: Request, res: Response) {
+  console.log('Processing [PUT /users/:user_id]...');
+  res.status(200).json(user_one);
+});
 
+app.delete('/users/:user_id', function (req: Request, res: Response) {
```

Comment:
Ah, I see why you need to call `json` here. A response needs to be ended: https://expressjs.com/en/api.html#res.end. Sending data implicitly does it for you.